ft-sys/src/diesel_sqlite/types/mod.rs

```diff
@@ -125,6 +125,20 @@ impl ToSql<sql_types::Double, Sqlite> for f64 {
     }
 }
 
+impl ToSql<sql_types::Jsonb, Sqlite> for serde_json::Value {
+    fn to_sql<'b>(&'b self, out: &mut Output<'b, '_, Sqlite>) -> serialize::Result {
+        let b = serde_json::to_vec(self)?;
+        out.set_value(b);
+        Ok(IsNull::No)
+    }
+}
+
+impl FromSql<sql_types::Jsonb, Sqlite> for serde_json::Value {
```

Comment:
Ideally we should implement `FromSql<sql_types::Jsonb, Sqlite>` and `ToSql<sql_types::Jsonb, Sqlite>` for any type `T` that implements `Serialize`/`Deserialize` traits.